agent/agent-tooling/src/main/java/com/microsoft/applicationinsights/agent/internal/exporter/AzureMonitorMetricExporter.java

```diff
@@ -119,7 +119,7 @@ private List<TelemetryItem> convertOtelMetricToAzureMonitorMetric(MetricData met
 
     for (PointData pointData : metricData.getData().getPoints()) {
       MetricTelemetryBuilder builder = telemetryClient.newMetricTelemetryBuilder();
-      builder.setTime(FormattedTime.offSetDateTimeFromEpochNanos(pointData.getStartEpochNanos()));
+      builder.setTime(FormattedTime.offSetDateTimeFromEpochNanos(pointData.getEpochNanos()));
```

Comment:
@heyams I think this is more expected to report the end time, and is consistent with what we do for micrometer pre-aggregated metrics